packages/headless/src/controllers/generated-answer/headless-generated-answer.ts

```diff
@@ -51,7 +58,20 @@ export interface GeneratedAnswer extends Controller {
 /**
  * @internal
  */
-export function buildGeneratedAnswer(engine: SearchEngine): GeneratedAnswer {
+export interface GeneratedAnswerProps {
+  /**
+   * The desired format to apply to generated answers when the controller first loads.
+   */
+  responseFormat: GeneratedResponseFormat;
```

Comment:
I have a similar requirement in my PR and defined [initialState](https://github.com/coveo/ui-kit/pull/3276/files#diff-40f9b6094183b67c164540da56e5747ca8dec42711e580891dd77cd7a682a008R63) to store this kind of information.

Reply:
Haha i commented the same thing on your PR, basically at the same time. I'll let yours go first and adapt mine after. Reviewing now